app/Http/Controllers/GistsController.php

```diff
@@ -62,6 +67,32 @@ public function show($username, $gistId)
 
         return View::make('gistlogs.show')
             ->with('gistlog', $gistlog)
-            ->with('pageTitle', $gistlog->title.' | '.$gistlog->author);
+            ->with('pageTitle', $gistlog->title.' | '.$gistlog->author)
```

Comment:
Spaces around `.` please -- can you install TLint on this project? Thanks!
```suggestion
            ->with('pageTitle', $gistlog->title . ' | ' . $gistlog->author)
```